app/src/main/java/com/craxiom/networksurveyplus/parser/QcdmLteParser.java

```diff
@@ -274,6 +325,7 @@ public static int getGsmtapLteRrcSubtype(int versionNumber, int channelType)
                 break;
 
             case 20: // 0x14
+            case 24: // 0x18
```

Comment:
Did you find any documentation on version number 24 (0x18) or how did you know that this is the right mapping?

Reply:
No document. In debug mode, my device is mapping to 24 (0x18), and most messages can be correctly decoded in WireShark